src/qiboml/backends/jax.py

```diff
@@ -132,23 +134,23 @@ def matrix_fused(self, fgate):
             num_controls = len(gate.control_qubits)
             if num_controls > 0:
                 gmatrix = self.jax.scipy.linalg.block_diag(
-                    self.np.eye(2 ** len(gate.qubits) - len(gmatrix)), gmatrix
+                    self.identity(2 ** len(gate.qubits) - len(gmatrix)), gmatrix
                 )
             # Kronecker product with identity is needed to make the
             # original matrix have shape (2**rank x 2**rank)
-            eye = self.np.eye(2 ** (rank - len(gate.qubits)))
-            gmatrix = self.np.kron(gmatrix, eye)
+            eye = self.identity(2 ** (rank - len(gate.qubits)))
+            gmatrix = self.kron(gmatrix, eye)
             # Transpose the new matrix indices so that it targets the
             # target qubits of the original gate
             original_shape = gmatrix.shape
-            gmatrix = self.np.reshape(gmatrix, 2 * rank * (2,))
+            gmatrix = self.eshape(gmatrix, 2 * rank * (2,))
```

Comment:
```suggestion
            gmatrix = self.reshape(gmatrix, 2 * rank * (2,))
```

Typo?

P.S.: Actually, I checked, and it's also in the linter